public/doc/openapi-3-v1.0.0.json

```diff
@@ -1531,6 +1531,9 @@
           "imported": {
             "type": "string"
           },
+          "source_ref": {
+            "type": "string"
+          },
```

Comment:
Can you use the openapi generator task? `bin/rake openapi:generate`

The content is correct but the order is out of place so it will look like more changes the next time we run the generator for something else